core/app/models/spree/adjustment.rb

```diff
@@ -177,11 +174,6 @@ def update!(target = nil)
 
     private
 
-    def update_adjustable_adjustment_total
-      # Cause adjustable's total to be recalculated
-      ItemAdjustments.new(adjustable).update
-    end
-
     def require_promotion_code?
       promotion? && source.promotion.codes.any?
     end
```

Comment:
Just wanted to point out this bad boy here:

https://github.com/solidusio/solidus/blob/master/core/app/models/concerns/spree/adjustment_source.rb#L7

It sort of relied on the "automatic"  after destroy of adjustments. Should we maybe add an `order.update!' to all those incomplete orders, for which we have deleted an adjustment?